bazel/src/test/data/googleapis/google/example/library/v1/BUILD.bazel.baseline

```diff
@@ -80,7 +80,7 @@ java_gapic_library(
 java_gapic_test(
     name = "library_java_gapic_test_suite",
     test_classes = [
-        "com.google.cloud.example.library.v1.LibraryServiceClientTest",
+        "com.google.example.library.v1.LibraryServiceClientTest",
```

Comment:
The actual clients (regardless if they are micro or macro generated) are still distributed under `"cloud"` namespace. Where does the "cloud" portion in the namespace should come from now, if not from gapic yaml? 

I'm afraid that we may still be generating client libraries under cloud namespace, but the tests specified in the build file may be loosing the cloud portion of the package, making it a broken test class name essentially.

Reply:
If you're referring to this Library client specifically, any other suggested value for `java_package` in the protos are welcome. This test means to exercise the "detect package from proto" case with a package distinct from the `gapic.yaml` one, so any value that achieves that is fine.

For other APIs, many have `.cloud` in their proto-specified `java_package` (e.g. [Functions](https://github.com/googleapis/googleapis/blob/master/google/cloud/functions/v1/functions.proto#L33)).